ui/raidboss/data/04-sb/raid/o2n.js

```diff
@@ -0,0 +1,411 @@
+'use strict';
+
+// O2N - Deltascape 2.0 Normal
+[{
+  zoneRegex: {
+    en: /^Deltascape \(V2\.0\)$/,
+  },
+  timelineFile: 'o2n.txt',
+  timelineTriggers: [
+    {
+      id: 'O2N Paranormal Wave',
+      regex: /Paranormal Wave/,
+      beforeSeconds: 5,
+      infoText: function(data) {
+        if (data.activeTank == data.me) {
+          return {
+            en: 'Tank cleave on YOU',
+          };
+        }
+        return {
+          en: 'Avoid tank cleave',
+        };
+      },
+    },
+  ],
+  triggers: [
+    {
+      // Used for avoiding Paranormal Wave.
+      id: 'O2N Tank Tracking',
+      regex: Regexes.ability({ id: '24E8', source: 'Catastrophe' }),
+      regexDe: Regexes.ability({ id: '24E8', source: 'Katastroph' }),
+      regexFr: Regexes.ability({ id: '24E8', source: 'Catastrophe' }),
+      regexJa: Regexes.ability({ id: '24E8', source: 'カタストロフィー' }),
+      regexCn: Regexes.ability({ id: '24E8', source: '灾变者' }),
+      regexKo: Regexes.ability({ id: '24E8', source: '카타스트로피' }),
+      run: function(data, matches) {
+        data.activeTank = matches.target;
+      },
+    },
+    {
+      id: 'O2N Gravitational Manipulation',
+      regex: Regexes.headMarker({ id: '0071' }),
+      condition: function(data) {
+        // The active tank shouldn't participate in the stack, as the boss would turn.
+        return !data.me == data.activeTank;
+      },
+      alertText: function(data, matches) {
+        if (data.me == matches.target) {
+          return {
+            en: 'Stack marker on YOU',
+          };
+        }
+        return {
+          en: 'Levitate--Stack on ' + data.shortName(matches.target),
```

Comment:
Maybe "Levitate and Stack on" etc vs the double dash? (Or alternatively use Responses.stackOn('alert') and then have a second "levitate" info callout for anybody not levitating?)

Reply:
That second one should be better. I was kind of trying to avoid overlapping callouts, but you're right, as it's written it calls too much if the user is already levitating.